openml/runs/functions.py

```diff
@@ -499,7 +499,7 @@ def _calculate_local_measure(sklearn_fn, openml_name):
 
         if isinstance(task, (OpenMLClassificationTask, OpenMLLearningCurveTask)):
 
-            for i, tst_idx in enumerate(test_indices):
+            for i, _ in enumerate(test_indices):
```

Comment:
Here you can (and should) keep the `tst_idx` as you're using it below.

Reply:
I [replaced](https://github.com/openml/openml-python/blob/da8dbb921e86c5ad5058d62dc12ad9d45fbfcd47/openml/runs/functions.py#L526) the usage of `tst_idx` with `test_indices[i]` to be consistent with the subsequent `elif` blocks where we are enumerating in a similar fashion of `for i, _ in enumerate(test_indices)`.

Is that alright? Or shall I replace it with `tst_idx` as it was, and push again?

Reply:
> Or shall I replace it with tst_idx as it was, and push again?

Yes, please. Please use `_` only if the variable is not used later on. Here it's used, so no need for an underscore. In the other two cases it's not used, so it should be `_`.

Reply:
Okay, done!